pyquil/quilbase.py

```diff
@@ -2695,21 +2837,33 @@ def qubit(self, qubit: QubitDesignator) -> None:
 
     @property
     def memory_reference(self) -> Optional[MemoryReference]:
+        """Get the memory reference this calibration writes to."""
         return MemoryReference._from_parameter_str(super().parameter)
 
     @memory_reference.setter
     def memory_reference(self, memory_reference: MemoryReference) -> None:
         quil_rs.MeasureCalibrationDefinition.parameter.__set__(self, memory_reference.name)  # type: ignore[attr-defined] # noqa
 
     @property
-    def instrs(self) -> List[AbstractInstruction]:
+    def instrs(self) -> list[AbstractInstruction]:
+        """Get the instructions in the calibration."""
         return _convert_to_py_instructions(super().instructions)
 
     @instrs.setter
-    def instrs(self, instrs: List[AbstractInstruction]) -> None:
+    def instrs(self, instrs: list[AbstractInstruction]) -> None:
         quil_rs.MeasureCalibrationDefinition.instructions.__set__(self, _convert_to_rs_instructions(instrs))  # type: ignore[attr-defined] # noqa
 
+    @property  # type: ignore[override]
+    def instructions(self) -> list[AbstractInstruction]:
```

Comment:
Was this added to restore some backwards compatibility?

Reply:
Ah, I had this change locally, probably shouldn't ship it with these changes.

For context, pyQuil historically uses the name `instrs`. In pyQuil v4, this class inherited its quil-rs counterpart, which uses the name `instructions`. Since the latter wasn't wrapped in any way, it returns quil-rs types. Some users reported confusion because they were accidentally using that property.

Reply:
removed